docs/flink_job_cluster_guide.md

```diff
@@ -0,0 +1,108 @@
+# Create Flink job clusters with Helm Chart
+
+## Overview
+
+This Helm Chart is an addition to the [existing](https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/tree/master/config/samples) way of deploying Flink job clusters.
+
+## Prerequisites
+
+
+- Fink Operator Image Version:  ```gcr.io/flink-operator/flink-operator:v1beta1-6``` follow these [instructions](https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/tree/master/helm-chart/flink-operator) to deploy the Operator
+- Flink Image Version: ```flink:1.9.3``` or ```flink:latest```
+- Helm Version 2.x or 3.x
+
+
+## Installing the Chart
+
+The instructions to install the Flink Job Cluster chart:
+
+1. Clone the repository to your local machine, which has access to your running kubernetes cluster.
+  ```bash
+  git clone https://github.com/GoogleCloudPlatform/flink-on-k8s-operator.git
+  ```
+2. Navigate to the following folder: ```/flink-on-k8s-operator/helm-chart```
+
+3. Use the following command to install the Flink job cluster chart:
+  ```bash
+  helm template --namespace example-namespace flink-job-cluster ./flink-job-cluster -f ./flink-job-cluster/values.yaml |kubectl apply -f -
```

Comment:
I got the following error about PodMonitor when running this command:

```
$ helm template --namespace foo flink-job-cluster ./flink-job-cluster -f ./flink-job-cluster/values.yaml | kubectl apply -f -

configmap/flink-job-cluster created
flinkcluster.flinkoperator.k8s.io/flink-job-cluster created
error: unable to recognize "STDIN": no matches for kind "PodMonitor" in version "monitoring.coreos.com/v1"
```

Reply:
True, I will add into the prerequisites that a PodMonitor will require you to have a prometheus-operator running in the cluster along with its custom resource definitions. I believe you did not have it active in your cluster?